test/libsolidity/ASTJSON/override.sol

```diff
@@ -1,13 +1,13 @@
 contract A {
```

Comment:
Why did you change this test?

Reply:
It was made of invalid code that for some reason still successfully exported ast json. However it was no longer successful after the contractDependencies change..

Reply:
I don't see why the dependencies change makes it invalid.

Reply:
Indeed, it seems and I was wrong. Those changes aren't really required, but having valid compilable code in a test still seems like a good idea to me.

Reply:
Can you extract it into its own PR?